service/frontend/workflowHandler.go

```diff
@@ -2343,6 +2343,10 @@ func (wh *WorkflowHandler) ListOpenWorkflowExecutions(ctx context.Context, reque
 		return nil, wh.error(errStartTimeFilterNotSet, scope)
 	}
 
+	if timestamp.TimeValue(request.GetStartTimeFilter().GetLatestTime()).IsZero() {
+		request.GetStartTimeFilter().LatestTime = timestamp.TimeNowPtrUtc()
```

Comment:
Time.max() instead of Time.now() because of possible clock skew?

Reply:
i wonder if it might also make sense
* to have the frontend just leave the values as is, and 
* to let whoever is actually handling the request to interpret IsZero values as "no upper bound", and produce the data accordingly.

Reply:
"Whoever" is persistence layer and we have 4 different calls to it. I will have to modify it in 4 places, or just one (here). I agree frontend handler is not a best place for it. Although all our validation/default logic is in frontend.